browser/brave_tab_helpers.cc

```diff
@@ -21,11 +21,13 @@
 #if defined(OS_ANDROID)
 #include "brave/browser/android/preferences/background_video_playback_tab_helper.h"
 #include "brave/browser/android/preferences/website/desktop_mode_tab_helper.h"
-#else
+#endif
+
 #if BUILDFLAG(BRAVE_REWARDS_ENABLED)
 #include "brave/browser/brave_rewards/rewards_tab_helper.h"
 #endif
 
+#if !defined(OS_ANDROID)
```

Comment:
this should not be added here and brave_stp_enabled defaults to `false` anyway so not sure why this would be necessary. If you're having problems you're probably missing a dep for `//brave/components/brave_shields/browser/buildflags` somewhere

Reply:
Yeah, not sure, probably it was some other headers there, but were removed during rebase. Anyways I removed that.